UnofficialSteamAuthenticator/Models/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Windows.UI.Xaml.Media.Imaging;
+using SteamAuth;
+
+namespace UnofficialSteamAuthenticator.Models
+{
+    /// <summary>
+    /// Type for showing users in a list
+    /// </summary>
+    internal class User : ModelBase
+    {
+        public ulong SteamId { get; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+        public BitmapImage Avatar { get; set; }
+        private readonly SteamGuardAccount _steamGuardAccount;
+
+        public User(ulong steamId, string title, string content, BitmapImage avatar)
+        {
+            SteamId = steamId;
+            _steamGuardAccount = Storage.GetSteamGuardAccount(steamId);
+
+            Title = title;
```

Comment:
Alright. The `this` qualifier was the same problem like the underscoring of private fields or direct vs suffix casting. The entire solution was just not consistent, so I decided to be consistent with the way I do it normally.

If you want to change it I can do it for the whole solution to be finally consistent :)